README.md

```diff
@@ -43,29 +43,101 @@ it('should demonstrate this matcher`s usage', async () => {
 > Note, you can also require `'jest-axe/extend-expect'` which will call `expect.extend` for you.
 > This is especially helpful when using the jest `setupTestFrameworkScriptFile` configuration.
 
-### With React
+### Testing React
 
 ```javascript
+const React = require('react')
+const App = require('./app')
+
 const { axe, toHaveNoViolations } = require('jest-axe')
+expect.extend(toHaveNoViolations)
+
+it('should demonstrate this matcher`s usage with react', async () => {
+  const results = await axe(<App/>)
+  expect(results).toHaveNoViolations()
+})
+```
+
+### Testing React with [Enzyme](https://airbnb.io/enzyme/)
+
+```javascript
+const React = require('react')
+const App = require('./app')
 
+const { mount } = require('enzyme')
+const { axe, toHaveNoViolations } = require('jest-axe')
 expect.extend(toHaveNoViolations)
 
+it('should demonstrate this matcher`s usage with enzyme', async () => {
+  const wrapper = mount(<App/>)
+  const results = await axe(container.getDOMNode())
+  
+  expect(results).toHaveNoViolations()

```

Comment:
This example should say:

```javascript
  const wrapper = mount(<App/>, { attachTo: document.body })
  const results = await axe(container.getDOMNode())
  
  expect(results).toHaveNoViolations()
  wrapper.detach()
```

Reply:
We don't need to use the `attachT` argument. This lib already takes care of this for us, and saves us having to clean up afterwards with `.detach()`

Reply:
You're not wrong, but I think `jest-axe` should avoid modifying the underlying document whenever possible. I think it's probably better to let people's test frameworks handle that part. The fact that `jest-axe` will put HTML into the document should probably be a sort of "fallback plan".

Reply:
Adding the `attachTo: document.body` causing a large warning during to testing: 

```
Warning: render(): Rendering components directly into document.body is discouraged,
since its children are often manipulated by third-party scripts and browser extensions.
This may lead to subtle reconciliation issues. Try rendering into a container element
created for your app.
```

Should we omit the `attachTo`, given then error?
Or should we go with something like: 
```js
const testContainer = document.createElement('div')
document.body.appendChild(testContainer)

const element = React.createElement('img', { src: '#' })
const wrapper = mount(element, { attachTo: testContainer })
const results = await axe(wrapper.getDOMNode())
```

Reply:
Eek, yeah, that's annoying. It's *probably* fine to omit the `attachTo`, but I'm a little worried about the fact that when you replace the `innerHTML`, it won't restore the event handlers.

I guess since `enzyme` and `vue` aren't actually rendering into the document by default anyway, it's probably fine that jest-axe modifies the document.body in those cases.

Reply:
Yeah, I agree. It's not ideal but given the current way they are handling rendering, I think it's ok. I've updated the README accordingly.